### ISSUE TITLE
Investigate GCS destination integration test failure

### ISSUE THREAD

**Issue by tuliren** [airbytehq/airbyte#9631]
The integration test passed locally, but failed on CI ([attempt](https://github.com/airbytehq/airbyte/actions/runs/1721221730/attempts/1)).

**Comment by tuliren** [comment]
The current hypothesis:
1. The sync command only needs 6 permissions, as listed in the [doc](https://docs.airbyte.com/integrations/destinations/gcs#setup-guide).
2. The connection check command, however, requires 2 more permissions (`multipartUpload.list` and `multipartUpload.listAll`), which is incorrect. So the check tests are failing in the acceptance test.

Next step:
1. Confirm the above hypothesis.
2. Fix the connection check command so that it does not need the two extra permission to pass.
3. If there is no way to fix the connection check, either remove the check that requires the extra permission, or list the two extra permission on the doc (not ideal).

**Comment by tuliren** [comment]
This issue has been fixed in version `0.1.21`. Now only the permissions listed in the [doc](https://docs.airbyte.com/integrations/destinations/gcs#setup-guide) are needed to sync data to GCS:

```
storage.multipartUploads.abort
storage.multipartUploads.create
storage.objects.create
storage.objects.delete
storage.objects.get
storage.objects.list
```
